electric_emission_cost/costs.py

```diff
@@ -301,7 +301,9 @@ def get_charge_dict(start_dt, end_dt, rate_data, resolution="15m"):
     return charge_dict
 
 
-def get_charge_df(start_dt, end_dt, rate_data, resolution="15m"):
+def get_charge_df(
+    start_dt, end_dt, rate_data, resolution="15m", keep_fixed_charges=False
```

Comment:
I think the default behavior should be to keep the fixed charges (i.e., `keep_fixed_charges =True` not `False`) but otherwise this seems like a good feature. Is there a reason to go through the effort of dividing the fixed charge over the length of the array? Maybe it could be useful and I've just never computed them in that way

Reply:
That makes sense! I figured it was either that or assign it to time step 0 and make the rest of the time steps 0 value. Either way, we need an array of length T to build the data frame.